website/versioned_docs/version-0.10.1/concepts.md

```diff
@@ -154,7 +154,7 @@ Following illustrates how the table works, and shows two types of queries - snap
 There are lot of interesting things happening in this example, which bring out the subtleties in the approach.
 
  - We now have commits every 1 minute or so, something we could not do in the other table type.
- - Within each file id group, now there is an delta log file, which holds incoming updates to records in the base columnar files. In the example, the delta log files hold
+ - Within each file id group, now there is an delta log file, which holds incoming updates to records in the row based files. In the example, the delta log files hold
```

Comment:
here I think the description is correct and why we need change it?

Reply:
I guess a better modification here would be - "which holds incoming updates to records already present in the base columnar files." 
That would clear the confusion which might have prompted you to raise this PR @lipsa308

Reply:
> I guess a better modification here would be - "which holds incoming updates to records already present in the base columnar files." That would clear the confusion which might have prompted you to raise this PR @lipsa308

This reads more clear.. @lipsa308 can you update as @pratyakshsharma suggested above? also can you pls update the current version docs? not just for 0.10.1

Reply:
@lipsa308 I addressed the comments for you.